hwsd/filter.cpp

```diff
@@ -135,7 +135,7 @@ class SessionFilter
 public:
     explicit SessionFilter( const std::string& name ) : name_( name ) {}
 
-    const std::string& name_;
+    const std::string name_;
```

Comment:
My question is whether this is the only instance of this bug or not. And the answer is no: NetFilter, at filters.cpp:226 has a similar bug.